### PR TITLE
Fix UI missing Brand colors (BASED ON #3703)

### PR DESCRIPTION
**Pull request by scouillard**
## Description

Fix some small issues regarding the Brand colors, mostly the "switch" and "nav-link".
Remove $primary and $primary-light variable from CSS.
Add $muted variable for bootstrap text-muted color.

## Screenshot
image

### CODE REVIEW

**Comment by scouillard** [comment]
I will spend some time to clean up our CSS files after the demo.
Need to find a way to scope our classes properly.

File: app/assets/stylesheets/application.bootstrap.scss [hunk 1]
```diff
@@ -146,3 +166,16 @@ input.search-bar {
     opacity: 0.8;
   }
 }
+
+.form-switch .form-check-input:focus {
```
Comment (KH-Amir-TN) [inline app/assets/stylesheets/application.bootstrap.scss:170]: I'm just worried about stepping on Bootstrap's toes (if he got some 😅) https://getbootstrap.com/docs/5.0/forms/checks-radios/

Comment (scouillard) [reply app/assets/stylesheets/application.bootstrap.scss:170]: Yeah changing the switch default color is super annoying for some reason.

**Review by farhatahmad — APPROVED** [review]

File: app/assets/stylesheets/application.bootstrap.scss (hunk 1 above)
Comment (KH-Amir-TN) [reply app/assets/stylesheets/application.bootstrap.scss:171]: We shouldn't be using raw bootstrap in GL3.
Those `form-switch` will be refactored to use `react-bootstrap` [CHECK HERE](https://react-bootstrap.netlify.app/forms/checks-radios/#switches).

Comment (scouillard) [reply app/assets/stylesheets/application.bootstrap.scss:171]: What are you linking to? It's the only way to remove the blue color om switches as far as I know. 

I don't think it's a big deal to overwrite bootstrap components colors. I would not overwrite paddings, widths, etc.

**Pull request merged**